ui-tests/src/test/java/org/openmrs/reference/FindPatientByNameTest.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.reference;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.page.*;
+import org.openmrs.uitestframework.test.TestBase;
+import org.openmrs.uitestframework.test.TestData;
+import org.junit.*;
```

Comment:
We do not encourage using wild card imports as per: https://wiki.openmrs.org/display/docs/Coding+Conventions